config/environment.default.js

```diff
@@ -10,10 +10,10 @@ module.exports = {
   // The REST API server settings.
   rest: {
     ssl: true,
-    host: 'dspace7.4science.it',
+    host: 'dspace7-entities.atmire.com',
     port: 443,
     // NOTE: Space is capitalized because 'namespace' is a reserved string in TypeScript
-    nameSpace: '/dspace-spring-rest/api'
+    nameSpace: '/rest/api'
```

Comment:
Like the comment I did in Dspace API PR (), in order for this to work by default, you need to change this setting to "**spring-rest**"

Reply:
@paulo-graca : Again, I agree for the final release we'll want this set to the name of the REST API webapp. However, again (as commented above) for testing purposes we are setting this to the path of the current REST API demo site by default.